### PR TITLE
Add the AMD64 QEMU target.

### PR DESCRIPTION
**Pull request by MichalBlk**
Supply `qemu-mimiker-x86_64`.

### CODE REVIEW

File: toolchain/qemu-mimiker/debian/changelog [hunk 1]
```diff
@@ -1,3 +1,9 @@
+qemu-mimiker (4.2.1+mimiker1) unstable; urgency=medium
```
Comment (cahirwpz) [inline toolchain/qemu-mimiker/debian/changelog:1]: Please bump up version.

File: toolchain/qemu-mimiker/debian/changelog [hunk 2]
```diff
@@ -1,3 +1,9 @@
...
+
+  * Add system simulator for AMD64.
+
+ -- Krystian Bacławski <cahirwpz@cs.uni.wroc.pl>  Fri, 26 Mar 2021 16:01:25 +0100
```
Comment (cahirwpz) [inline toolchain/qemu-mimiker/debian/changelog:5]: Please update date with output from `date -R` command.

**Review by cahirwpz — CHANGES_REQUESTED** [review]

File: toolchain/qemu-mimiker/debian/changelog (hunk 1 above)
Comment (MichalBlk) [reply toolchain/qemu-mimiker/debian/changelog]: Done.

File: toolchain/qemu-mimiker/debian/changelog (hunk 2 above)
Comment (MichalBlk) [reply toolchain/qemu-mimiker/debian/changelog]: Done.

**Review by cahirwpz — APPROVED** [review]

**Pull request merged**